sdv/metadata/utils.py

```diff
@@ -150,3 +152,26 @@ def strings_from_regex(regex, max_repeat=16):
             sizes.append(size)
 
     return _from_generators(generators, max_repeat), np.prod(sizes)
+
+
+def read_json(filepath):
+    """Validate and open a file path."""
+    filepath = Path(filepath)
+    if not filepath.exists():
+        raise ValueError(
+            f"A file named '{filepath.name}' does not exist. "
+            'Please specify a different filename.'
+        )
+
+    with open(filepath, 'r', encoding='utf-8') as metadata_file:
+        return json.load(metadata_file)
+
+
+def validate_path(filepath):
+    """Validate a file path."""
```

Comment:
Can we rename this and change the description to be more descriptive like `validate_file_does_not_exist`